aiohttp/multidict.py

```diff
@@ -305,6 +305,9 @@ def __init__(self, items):
     def __len__(self):
         return len(self._items)
 
+    def __repr__(self):
```

Comment:
Good question. I suggest pushing along with `test___repr__`: https://github.com/KeepSafe/aiohttp/blob/master/tests/test_multidict.py#L269

Maybe I should refactor multidicts' test hierarchy.